src/main/java/org/tron/core/capsule/StorageRowCapsule.java

```diff
@@ -39,15 +38,18 @@ private void markDirty() {
     dirty = true;
   }
 
-  private StorageRowCapsule() {
-  }
-
   public StorageRowCapsule(byte[] rowKey, byte[] rowValue) {
     this.rowKey = rowKey;
     this.rowValue = rowValue;
     markDirty();
   }
 
+  public StorageRowCapsule(StorageRowCapsule rowCapsule) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this constructor to comply with Java Code Conventions. ([squid:S1213](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1213))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6rASgN4uCclo1g8&open=AWbJU6rASgN4uCclo1g8)

[ISSUE_KEY]: AWbJU6rASgN4uCclo1g8